queries/synths/useFeeReclaimPeriodsQuery.ts

```diff
@@ -50,10 +50,9 @@ const useFeeReclaimPeriodsQuery = (
 			}
 
 			const waitingPeriodsRaw = (await ethcallProvider.all(
-				waitingPeriodCalls,
-				{}
+				waitingPeriodCalls
 			)) as ethers.BigNumberish[];
-			const feesRaw = await ethcallProvider.all(feeCalls, {});
+			const feesRaw = (await ethcallProvider.all(feeCalls)) as [any, any, any];
```

Comment:
Not sure if this type is correct. `settlementOwing` returns a tuple of three `uint` values. I think this should be:

```suggestion
			const feesRaw = (await ethcallProvider.all(feeCalls)) as [ethers.BigNumberish, ethers.BigNumberish, ethers.BigNumberish][];
```

Reply:
> Not sure if this type is correct. `settlementOwing` returns a tuple of three `uint` values. I think this should be:

this does not work.  tried it, and sub requires a number type, and throws an error on ethers.BigNumberish.  I understand the desire to type this, but whatever is coming back needs iterable, so it's not a Number either.  We can create a ticket to fix this later, if we want.  I've added a TODO for now